### PR TITLE
Added button functionality to the header

### PR DESCRIPTION
**Pull request by JohanLaubscher**
Added button hover css to the header and the redirect of the login and settings

### CODE REVIEW

File: ChatterBox/try-app/src/container/Header/index.js [hunk 1]
```diff
@@ -1,17 +1,46 @@
 import React from 'react'
-import FlatButton  from 'material-ui/FlatButton';
+import { Route, Link } from 'react-router-dom'
+import { push } from 'react-router-redux'
+import { bindActionCreators } from 'redux'
+import { connect } from 'react-redux'
+import FlatButton from 'material-ui/FlatButton';
```
Comment (grootendorste) [inline ChatterBox/try-app/src/container/Header/index.js]: Consistency, decide if the semi-colon is needed or not then use it everywhere.

Comment (grootendorste) [reply ChatterBox/try-app/src/container/Header/index.js]: This applies from now on.

File: ChatterBox/try-app/src/container/Header/index.js [hunk 2]
```diff
@@ -1,17 +1,46 @@
...
-   <div class="pull-right"><span class="user-name">Johan Laubscher</span><img id="profile-img" class="profile-img" src="//ssl.gstatic.com/accounts/ui/avatar_2x.png" /><img id="profile-img" class="profile-img" src="//ssl.gstatic.com/accounts/ui/avatar_2x.png" /></div>
- </div>
-</div>
+  <div class="header">
+    <div class="left-buttons">
+      <div class="dropdown">
+        <button class="dropbtn btn header-button"><span class="button-label">NEW CHAT</span></button>
+        <div class="dropdown-content">
+          <TextField
+            hintText="Friends Email"
```
Comment (grootendorste) [inline ChatterBox/try-app/src/container/Header/index.js]: Do you mean "Friend's Email"?

File: ChatterBox/try-app/src/container/Header/index.js (hunk 1 above)
Comment (JohanLaubscher) [reply ChatterBox/try-app/src/container/Header/index.js]: Cool, will keep in mind.

File: ChatterBox/try-app/src/container/Header/index.js (hunk 2 above)
Comment (JohanLaubscher) [reply ChatterBox/try-app/src/container/Header/index.js]: Yes, will change it now.

File: ChatterBox/try-app/src/container/Header/index.js (hunk 1 above)
Comment (jaderabbit) [reply ChatterBox/try-app/src/container/Header/index.js]: @grootendorste This is an external package They can't chose it

File: ChatterBox/try-app/src/index.css [hunk 3]
```diff
@@ -1,244 +1,392 @@
...
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: sans-serif;
+  background-color:  #004853;
+}
+
+.login{
+  height:644px;width:533px;border-radius:12px;background-color:#FDFDFD;margin-right: 454px; margin-left: 454px; margin-top: 174px; margin-bottom: 174px;
```
Comment (jaderabbit) [inline ChatterBox/try-app/src/index.css]: Each on a new line pls. This is nonsense :P

Comment (jaderabbit) [reply ChatterBox/try-app/src/index.css]: fix throughout

File: ChatterBox/try-app/src/index.css [hunk 4]
```diff
@@ -1,244 +1,392 @@
...
+      width: 126px;
+      color: #FFFFFF;
+      font-family: helvetica;
+      font-size: 18px;
+      font-weight: 500;
+      line-height: 25px;
+      text-align: center;
+    }
+
+
```
Comment (jaderabbit) [inline ChatterBox/try-app/src/index.css]: whitespace for what?

File: ChatterBox/try-app/src/index.css [hunk 5]
```diff
@@ -1,244 +1,392 @@
...
+      font-weight: 500;
+      line-height: 25px;
+      text-align: center;
+    }
+
+    
+
+
+
+    .dropbtn {
```
Comment (jaderabbit) [inline ChatterBox/try-app/src/index.css]: fix indentation. ugly!

File: ChatterBox/try-app/src/index.css [hunk 6]
```diff
@@ -1,244 +1,392 @@
...
+  padding-right: 55px;
+}
+
+.txtbox{
+  padding-left: 72px;
+  padding-right: 72px;
+  margin-top: 20px;
+}
+
+.login-2 {
```
Comment (jaderabbit) [inline ChatterBox/try-app/src/index.css]: login-2? What is this? Better names

File: ChatterBox/try-app/src/index.css [hunk 7]
```diff
@@ -1,244 +1,392 @@
...
+  align-content: center;
+  border-radius: 200px !important;
+  border: solid !important;
+  border-color: #FFFFFF !important;
+  border-width: 100px !important;
+  margin-left: 55px !important;
+  color: #979797 !important;
+}
+
+.btnStepTwo{
```
Comment (jaderabbit) [inline ChatterBox/try-app/src/index.css]: bad name

File: ChatterBox/try-app/src/container/Header/index.js [hunk 8]
```diff
@@ -1,17 +1,46 @@
...
+      <div class="pull-right">
+        <div class="account-dropdown">
+          <span class="account-dropbtn user-name">Raving Rabbits</span>
+          <div class="account-dropdown-content">
+            <a href="/settings">Settings</a>
+            <a href="/login">Log Out</a>
+          </div>
+        </div>
+        <img id="profile-img" class="profile-img" src="//ssl.gstatic.com/accounts/ui/avatar_2x.png" />
+        <img id="profile-img" class="profile-img" src="//ssl.gstatic.com/accounts/ui/avatar_2x.png" />
```
Comment (wscheffer) [inline ChatterBox/try-app/src/container/Header/index.js]: Haven't seen the design but should there be two of the same `img` tags?

Comment (JohanLaubscher) [reply ChatterBox/try-app/src/container/Header/index.js]: Yep, need to replace one with the user profile img and the other will be the icon of the application.

File: ChatterBox/try-app/src/index.css [hunk 9]
```diff
@@ -1,244 +1,392 @@
...
-      width: 126px;
-      color: #FFFFFF;
-      font-family: helvetica;
-      font-size: 18px;
-      font-weight: 500;
-      line-height: 25px;
-      text-align: center;
-    }
\ No newline at end of file
+
```
Comment (wscheffer) [inline ChatterBox/try-app/src/index.css]: Whats with the inconsistent formatting on this file?
VS Code has the ability to auto-format documents

File: ChatterBox/try-app/src/index.css [hunk 10]
```diff
@@ -1,244 +1,392 @@
...
+}
+
+.login{
+  height:644px;width:533px;border-radius:12px;background-color:#FDFDFD;margin-right: 454px; margin-left: 454px; margin-top: 174px; margin-bottom: 174px;
+}
+
+.welcome-to-the {	height: 67px;	
+  width: 424px;	
+  color: #004853;	
+  font-family: "helvetica";
```
Comment (wscheffer) [inline ChatterBox/try-app/src/index.css]: When using only the name of the font the style should be declared as `font-family: Helvetica;` also the `h` should be an `H`

File: ChatterBox/try-app/src/index.css [hunk 11]
```diff
@@ -1,244 +1,392 @@
...
+    height: 27px;	
+    width: 100%;	
+    color: #FFFFFF;	
+    font-family: "helvetica";	
+    font-size: 20px;	
+    line-height: 27px;	
+    text-align:"center";}
+
+.btnLogin{	
+    position: absolute !important;
```
Comment (wscheffer) [inline ChatterBox/try-app/src/index.css]: Why are there so many `!important` in this css file?

Comment (JohanLaubscher) [reply ChatterBox/try-app/src/index.css]: Did not write that css, but will fix it now.

File: ChatterBox/try-app/src/index.css [hunk 12]
```diff
@@ -1,244 +1,319 @@
...
+  width: 533px;
+  border-radius: 12px;
+  background-color: #01B9BD;
+  margin-right: 454px;
+  margin-left: 454px;
+  margin-top: 174px;
+  margin-bottom: 174px;
+}
+
+.login-2 {
```
Comment (jaderabbit) [inline ChatterBox/try-app/src/index.css]: login-2? y tho?

Comment (JohanLaubscher) [reply ChatterBox/try-app/src/index.css]: Hi Jade, we had a problem with a merge where we included all text of both files. There was a lot of duplicates etc which I fixed, however I don't know where these css is being used. Should I just change the names and fix everything or should each team member fix their own styles? 
image

Comment (jaderabbit) [reply ChatterBox/try-app/src/index.css]: @JohanLaubscher Change and fix everything. You're a team :)

Comment (JohanLaubscher) [reply ChatterBox/try-app/src/index.css]: Cool, will do

File: ChatterBox/try-app/src/container/Registration/register.js [hunk 13]
```diff
@@ -56,3 +58,87 @@ export default connect(
 )(RegisterPage)
 
 
+
+
+// export default class registerUser extends Component{
```
Comment (jaderabbit) [inline ChatterBox/try-app/src/container/Registration/register.js]: remove commented code

File: ChatterBox/try-app/src/App.js [hunk 14]
```diff
@@ -1,19 +1,17 @@
...
 class App extends Component {
   render() {
     return (
       <div>
-        <main>
-          <Route exact path="/" component={Sidemenu} />
-        </main>
-      </div>    
-      );
+      <main>
```
Comment (jaderabbit) [inline ChatterBox/try-app/src/App.js]: indents?

File: ChatterBox/try-app/src/container/Registration/register.js [hunk 15]
```diff
@@ -38,6 +38,8 @@ const RegisterPage = props => (
     label="NEXT STEP" 
     labelColor="#FFFFFF"
     onClick={() => props.changePage()}
+    //onClick={this.userRegister}
```
Comment (jaderabbit) [inline ChatterBox/try-app/src/container/Registration/register.js]: remove commented code

**Pull request closed without merging**